backends/automl-go/components/graceful.go

```diff
@@ -0,0 +1,318 @@
+/*
```

Comment:
Can we import the lib with the package manager instead of cloning the source code?